Automattic/jetpack
# Load Scheduled_Updates for non-wpcom users

Fixes https://github.com/Automattic/automattic-for-agencies-dev/issues/906

## Proposed changes:

* This was introduced by this PR: https://github.com/Automattic/jetpack/pull/38419
  * Moving the `Scheduled_Updates` initialization from `load_features` to `load_wpcom_user_features` made this module not be initialized when a site has the `is_fully_managed_agency_site` option (the `is_wpcom_user` function checks for this option).
* The `Scheduled_Updates` uses a hook to add an extra field `is_managed` for each plugin through `add_is_managed_extension_field` on `projects/packages/scheduled-updates/src/class-scheduled-updates.php`
  * So, if the module is missing for a site, this field won't be returned on `/plugins` request
* Moving just the `Scheduled_Updates` initialization back to the `load_features` fixes the issue 

### Other information:


## Jetpack product discussion



Slack Thread: p1722965693473719-slack-C02TCEHP3HA

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create two testing sites through A4A (one with the WPcom features enabled and the other disabled)
  * Meaning `is_fully_managed_agency_site` true for one site and false for the other
* [Follow the instructions here to setup your sites](https://github.com/Automattic/jetpack/pull/38772#issuecomment-2274322170)
* Go to https://wordpress.com/plugins/scheduled-updates/create
* Select the managed site, and it should load only the non-managed plugins
* Select the unmanaged site, and similarly, it should load only the non-managed plugins
* Check if the `/plugins` request returns the `is_managed` value for each plugin in the response
